modules/opensearch/opensearch.client.inc

```diff
@@ -561,6 +561,9 @@ function _opensearch_predict_cache(array $collections, TingClientRequest $reques
  */
 function _opensearch_cache(TingClientRequest $request, $response = NULL) {
   static $static_cache;
```

Comment:
Is it not possible to simply do:

`static $static_cache = array();`

http://php.net/manual/en/language.variables.scope.php#language.variables.scope.static see under "Example #7 Declaring static variables"

Reply:
Or "Example #5 Example use of static variables"